apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -97,6 +94,7 @@ const ScheduleDirection = ({
   stops,
   today,
   scheduleNote,
+  // This references the current bus variant, if applicable
```

Comment:
@hannahpurcell does this comment indicate that the variable name should be changed to be more descriptive?

Reply:
You know what?  YES!  `busVariantId` it is